modules/services/model/src/main/java/au/com/shiftyjelly/pocketcasts/models/entity/Bookmark.kt

```diff
@@ -0,0 +1,30 @@
+package au.com.shiftyjelly.pocketcasts.models.entity
+
+import androidx.room.ColumnInfo
+import androidx.room.Entity
+import androidx.room.Index
+import androidx.room.PrimaryKey
+import java.io.Serializable
+import java.util.Date
+
+@Entity(
+    tableName = "bookmarks",
+    indices = [
+        Index(name = "bookmarks_podcast_uuid", value = arrayOf("podcast_uuid"))
+    ]
+)
+data class Bookmark(
```

Comment:
Should we add a bookmark title field?

Reply:
Oh, I missed that one. I have added this column.